.github/workflows/changelog-draft,yml

```diff
@@ -0,0 +1,19 @@
+name: Release Drafter
+
+on:
+  push:
```

Comment:
Better to make it a manual trigger maybe?

Reply:
Auto-generation is actually handy